src/main.cpp

```diff
@@ -31,183 +31,224 @@ using TIME = NDTime;
+        cout << argv[0] << " path to the input file " << endl;
+        return 1; 
   }
 
   auto start = hclock::now(); //to measure simulation execution time
 
 /*************** Loggers *******************/
-  static std::ofstream out_data("../data/abp_output.txt");
-    struct oss_sink_provider{
-        static std::ostream& sink(){          
-            return out_data;
-        }
-    };
-
-using info=cadmium::logger::logger<cadmium::logger::logger_info, cadmium::dynamic::logger::formatter<TIME>, oss_sink_provider>;
-using debug=cadmium::logger::logger<cadmium::logger::logger_debug, cadmium::dynamic::logger::formatter<TIME>, oss_sink_provider>;
-using state=cadmium::logger::logger<cadmium::logger::logger_state, cadmium::dynamic::logger::formatter<TIME>, oss_sink_provider>;
-using log_messages=cadmium::logger::logger<cadmium::logger::logger_messages, cadmium::dynamic::logger::formatter<TIME>, oss_sink_provider>;
-using routing=cadmium::logger::logger<cadmium::logger::logger_message_routing, cadmium::dynamic::logger::formatter<TIME>, oss_sink_provider>;
-using global_time=cadmium::logger::logger<cadmium::logger::logger_global_time, cadmium::dynamic::logger::formatter<TIME>, oss_sink_provider>;
-using local_time=cadmium::logger::logger<cadmium::logger::logger_local_time, cadmium::dynamic::logger::formatter<TIME>, oss_sink_provider>;
-using log_all=cadmium::logger::multilogger<info, debug, state, log_messages, routing, global_time, local_time>;
-
-using logger_top=cadmium::logger::multilogger<log_messages, global_time>;
-
-
-/*******************************************/
-
-
-
-/********************************************/
-/****** APPLICATION GENERATOR *******************/
-/********************************************/
-string input_data_control = argv[1];
-const char * i_input_data_control = input_data_control.c_str();
-
-std::shared_ptr<cadmium::dynamic::modeling::model> generator_con = cadmium::dynamic::translate::make_dynamic_atomic_model<ApplicationGen, TIME, const char* >("generator_con" , std::move(i_input_data_control));
-
-
-/********************************************/
-/****** SENDER *******************/
-/********************************************/
-
-std::shared_ptr<cadmium::dynamic::modeling::model> sender1 = cadmium::dynamic::translate::make_dynamic_atomic_model<Sender, TIME>("sender1");
-
-/********************************************/
-/****** RECIEVER *******************/
-/********************************************/
-
-std::shared_ptr<cadmium::dynamic::modeling::model> receiver1 = cadmium::dynamic::translate::make_dynamic_atomic_model<Receiver, TIME>("receiver1");
-
+  static std::ofstream out_data("data/abp_output.txt");
+  struct oss_sink_provider{
+      static std::ostream& sink(){          
+          return out_data;
 
+        }
 
-/********************************************/
-/****** SUBNET *******************/
-/********************************************/
+    };
```

Comment:
In above lines check for indentation
Also curly brackets should end in the same column.

Reply:
Done as requested, Thanks